documents/charter.md

```diff
@@ -25,6 +25,8 @@ The properies of this mechanism are as follows:
 The network properties must be associated with a given application traversing the network, for example a video playback.
 1. Client initiation.
 The communication channel is initiated by a client device, just as the end to end application flows are also typically initiated by a client.
+1. Network to client. 
+The network provides the properties to the client. Other modes of operation (for example, a client providing properties to the network) are out of scope. 
```

Comment:
Seems like we could just delete the 2nd sentence on the line above

Reply:
That works for me!